tutorial/4. multiple_grid_cells_and_hypercube.ipynb

```diff
@@ -0,0 +1,1231 @@
+       "      <td>3.5798805150181336</td>\n",
+       "      <td>5.146485045328571</td>\n",
+       "      <td>8.378490527758816</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>4.0</td>\n",
+       "      <td>309.78338149363844</td>\n",
+       "      <td>101543.77643542801</td>\n",
+       "      <td>39.42402937477069</td>\n",
+       "      <td>3.513198536452761</td>\n",
+       "      <td>5.116679174625489</td>\n",
+       "      <td>8.474978377027266</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>...</th>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>96</th>\n",
+       "      <td>96.0</td>\n",
+       "      <td>309.78338149363844</td>\n",
+       "      <td>101543.77643542801</td>\n",
+       "      <td>39.42402937477069</td>\n",
+       "      <td>1.1449152016456725e-09</td>\n",
+       "      <td>2.669502749126976e-08</td>\n",
+       "      <td>17.104856060265632</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>97</th>\n",
+       "      <td>97.0</td>\n",
+       "      <td>309.78338149363844</td>\n",
+       "      <td>101543.77643542801</td>\n",
+       "      <td>39.42402937477069</td>\n",
+       "      <td>7.255822352343744e-10</td>\n",
+       "      <td>1.724905104014427e-08</td>\n",
+       "      <td>17.10485607013094</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>98</th>\n",
+       "      <td>98.0</td>\n",
+       "      <td>309.78338149363844</td>\n",
+       "      <td>101543.77643542801</td>\n",
+       "      <td>39.42402937477069</td>\n",
+       "      <td>4.576760504141316e-10</td>\n",
+       "      <td>1.1091297770220084e-08</td>\n",
+       "      <td>17.104856076556583</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>99</th>\n",
+       "      <td>99.0</td>\n",
+       "      <td>309.78338149363844</td>\n",
+       "      <td>101543.77643542801</td>\n",
+       "      <td>39.42402937477069</td>\n",
+       "      <td>2.8733459694094004e-10</td>\n",
+       "      <td>7.097135077178593e-09</td>\n",
+       "      <td>17.10485608072109</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>100</th>\n",
+       "      <td>100.0</td>\n",
+       "      <td>309.78338149363844</td>\n",
+       "      <td>101543.77643542801</td>\n",
+       "      <td>39.42402937477069</td>\n",
+       "      <td>1.795460281048044e-10</td>\n",
+       "      <td>4.519272406440178e-09</td>\n",
+       "      <td>17.10485608340674</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "<p>101 rows × 7 columns</p>\n",
+       "</div>"
+      ],
+      "text/plain": [
```

Comment:
You may want to mention that this output is for the first grid cell only, based on your index choice?